api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -46,10 +46,23 @@ message ExtAuthz {
 // in the authorization response should be sent to the upstream. See *allowed_authorization_headers*
 // bellow.
 //
-// A failed check will cause this filter to close the HTTP request normally with 403 (Forbidden),
+// A failed check will cause this filter to close the HTTP request with 403 (Forbidden),
 // unless a different status code has been indicated by the authorization server via response
-// headers. If other headers in the authorization response need to be sent to client, this can also
-// be done by specifying them in *allowed_authorization_headers*.
+// headers.
+//
+// If an error happens during the checking process, two situations may occur depending on the
+// filter's configuration:
+//
+//  1. When *failure_mode_allow* is true, traffic will be allowed at the presence of an error. This
```

Comment:
Nit: can you s/at the presence/in the presence/ in both these paragraphs?

Reply:
👍